load/UpdateDBPLanguageTranslation.py

```diff
@@ -52,40 +52,37 @@ def _processLanguageWithAltName(self, languageSourceId, languageRecord):
 
     def _updateOrInsertLanguageTranslations(self, languageNames, languageSourceId, languageTranslationId, priority):
         for languageName in languageNames:
-            insertRows = []
-            updateRows = []
+            if languageName != None and languageName != "":
+                insertRows = []
+                updateRows = []
 
-            indexedKey = "%s%s%s" % (languageSourceId,languageName,priority)
+                indexedKey = "%s%s%s" % (languageSourceId,languageName.lower(),priority)
```

Comment:
we have updated the way to index the language name because language name will be case insensitive.

Reply:
@bradflood ok I got it.

Brad this line won't alter the languageName when we store it into database. This line is storing into indexed array an flag to avoid to create two inserts when etl extracts from altName two similar languages but that one letter is uppercase for the first language and one letter is lowercase for the second one. E.g.

```sql
AltName = OluHaya | Oluhaya

OluHaya -> Uppercase H
Oluhaya -> lowercase h
```

If we don't store the index in lower etl would create:

```sql

INSERT INTO language_translations (name, priority, language_source_id, language_translation_id) VALUES ('OluHaya', '0', '2322', '8012');
INSERT INTO language_translations (name, priority, language_source_id, language_translation_id) VALUES ('Oluhaya', '0', '2322', '8012');
```
About the above inserts the second one would throw error because collation is case insensitive.